packages/jest-snapshot/src/SnapshotFile.js

```diff
@@ -31,8 +31,10 @@ class SnapshotFile {
     this._filename = filename;
     if (this.fileExists(filename)) {
       this._content = JSON.parse(fs.readFileSync(filename));
+      this._uncheckedKeys = new Set(Object.keys(this._content));
```

Comment:
those are neither removed nor invalid, this is the list of  keys that we didn't check yet for existence